docs/guides/references/configuration.mdx

```diff
@@ -124,7 +124,7 @@ For more options regarding screenshots, view the
 | Option                  | Default          | Description                                                                                                                                                                                                                                                                                                              |
 | ----------------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `trashAssetsBeforeRuns` | `true`           | Whether Cypress will trash assets within the `downloadsFolder`, `screenshotsFolder`, and `videosFolder` before tests run with `cypress run`.                                                                                                                                                                             |
-| `videoCompression`      | `32`             | The quality setting for the video compression, in Constant Rate Factor (CRF). The value can be `false` to disable compression or a value between `0` and `51`, where a lower value results in better quality (at the expense of a higher file size).                                                                     |
+| `videoCompression`      | `false`          | The quality setting for the video compression, in Constant Rate Factor (CRF). The value can be `false` to disable compression or a value between `0` and `51`, where a lower value results in better quality (at the expense of a higher file size).                                                                     |
```

Comment:
I think this should also mention that `true` is valid and sets the value to 32 CRF.

Reply:
This is handled in the `release-12.15.0` [branch](https://github.com/cypress-io/cypress-documentation/blob/release-12.15.0/docs/guides/references/configuration.mdx?plain=1#L127). It just hasn't made its way into `main` yet. Once released, we will need to merge `main` into the `v13` docs branch to make sure the language makes sense